effortless-technologies/elt-marketplace-wp
# [WIP] Implement checkout process integrating both effortless and amazon fulfilled products/services

This implements a multistep checkout process that integrates woozone and woocommerce to play 'nicely' together so a user can complete the entire checkout process in an intuitive, tasteful manner. 

**Testing**
+ Add amazon affiliate products to cart
+ Add effortless products to cart
+ Checkout as you would any other site
+ Should be able to purchase both effortless and amazon product with ease
